MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -111,29 +112,33 @@ def fetch_github_issues(args, db):
 
 			#Get the last updated time for the repo
 			since_info = get_collection_record({'repository': repo}, None, 'LastUpdatedInfo', db)
-			
-			num_issues = 0
 			since = None
 
-			#Get the last_updated time and total_issues_collected if the repo is found
+			#Get the last_updated time if the repo is found
 			if since_info:
 				since = since_info['last_updated_time']
-				num_issues = since_info['total_issues_collected']
-	
+			
 			pprint(f"Fetching open issues for {repo}")
 
-			num_issues += fetch_new_issues(repo, since, auth_token, db)
+			fetch_new_issues(repo, since, auth_token, db)
 
+			num_open_issues = db['Issues'].count_documents({'state': 'open'})
```

Comment:
`count_documents` will counts both new and existing data, and will re-count every time we collect new issues. This will cost longer when we get more collected data. 
Could you try to calculate them still in `fetch_new_issues`, but `plus` or `minus` issues numbers when store in DB (e.g. if a issue already in DB with `open` state, but got `closed` in new fetch, then `-1` for open issues, and similar for `reopen` issues and new `open` issues). Thanks.

Reply:
Done! Please check it and let me know if the new implementation looks better. I have attempted to accurately update the old values.